libs/community/langchain_community/vectorstores/clickhouse.py

```diff
@@ -209,6 +199,41 @@ def __init__(
         self.client.command(f"SET allow_experimental_{self.config.index_type}_index=1")
```

Comment:
@mneedham just confirming - I'm pretty sure this change turns this into ```self.client.command(f"SET allow_experimental_None_index=1")``` unless I'm mistaken. This seems undesirable?